matrix-meetings-bot/src/dto/MeetingCreateDto.ts

```diff
@@ -101,6 +103,14 @@ export class MeetingCreateDto {
   @IsOptional()
   participants?: ParticipantDto[];
 
+  /**
+   * power level required to send events in the room
+   */
+  @IsNumber({ maxDecimalPlaces: 0 })
+  @Min(0)
+  @IsOptional()
+  messaging_power_level?: number;
```

Comment:
We should also update the API definition in the https://github.com/nordeck/matrix-meetings/blob/main/docs/data-model.md file. Both for the event schema and the REST API endpoint.

Reply:
thanks for finding, this PR updates the API: https://github.com/nordeck/matrix-meetings/pull/201